### PR TITLE
Accept File in AirplaneFile constructor

### PR DESCRIPTION
**Pull request by colinking**
From testing Views locally with the latest commit of JS SDK, I noticed we weren't handling `File` types correctly. The `isBlob` check wasn't passing for `File` (which extends `Blob`).

Resolves TSK-1280

### CODE REVIEW

**Comment by colinking** [comment]
Current dependencies on/for this PR:
* main
  * **PR #233** <a href="https://app.graphite.dev/github/pr/airplanedev/node-sdk/233" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/airplanedev/node-sdk/233?utm_source=stack-comment).

File: packages/airplane/internal/file/file.ts [hunk 1]
```diff
@@ -185,9 +185,20 @@ export class AirplaneFile {
...
  * @see https://developer.mozilla.org/en-US/docs/Web/API/Blob
  */
 const isBlob = (b: unknown): b is Blob => {
-  return b != null && typeof b === "object" && b.constructor.name === "Blob";
+  if (b == null || typeof b !== "object") {
+    return false;
+  }
+
+  // Fallback to checking the object's StringTag to handle cases where Blob is not polyfilled.
+  if (("Blob" in globalThis && b instanceof globalThis.Blob) || String(b) === "[object Blob]") {
```
Comment (geodn) [inline packages/airplane/internal/file/file.ts:197]: I think the test passes with `b.constructor.name === "Blob"` as well as `String(b) === "[object Blob]"`. Any reason to prefer one over the other?

**Review by geodn — APPROVED** [review]

File: packages/airplane/internal/file/file.ts (hunk 1 above)
Comment (colinking) [reply packages/airplane/internal/file/file.ts:197]: The former didn't pass with `fetch-blob`. I believe their constructor name is `_Blob`. The latter checks `Symbol.StringTag` which should be more reliable!

Comment (colinking) [reply packages/airplane/internal/file/file.ts:197]: The latter checks `Symbol.StringTag` which should be more reliable than the constructor name.

**Pull request merged**